extensions/catalog/federated-catalog-cache/build.gradle.kts

```diff
@@ -29,6 +29,10 @@ dependencies {
     // generates random names
     implementation("info.schnatterer.moby-names-generator:moby-names-generator:20.10.0-r0")
 
+    // generates random names
+    // fixme: remove before PR submit
+    implementation("info.schnatterer.moby-names-generator:moby-names-generator:20.10.0-r0")
```

Comment:
Should this be removed before the merge?

Reply:
I kind of have grown to love those funny names :)
yeah, I'll remove it... :(